### ISSUE TITLE
[🐛 Bug]: Cannot resolve Selenium Java Bindings artifacts as modules (JPMS) in JDK 11

### ISSUE THREAD

**Issue by tarunaggarwal23** [SeleniumHQ/selenium#12089]
### What happened?

Java: OpenJDK 11

Selenium: 4.6.0

Maven: 3.5.2

Eclipse: 2022-12 (4.26.0)

 

I am getting this error: "cannot be resolved to a module" for the following lines:

                requires selenium.api;

                requires selenium.firefox.driver;

                requires selenium.remote.driver;

                requires selenium.ie.driver;

                requires selenium.edge.driver;

                requires selenium.chrome.driver;

Deleting module-info.Java resolves all errors.

### How can we reproduce the issue?

```shell
mvn -U -X clean install -Dmaven.test.skip=true -Dmaven.wagon.http.ssl.insecure=true
```

### Relevant log output

```shell
module not found: selenium.api

module not found: selenium.remote.driver
```

### Operating System

Windows 10

### Selenium version

4.6.0

### What are the browser(s) and version(s) where you see this issue?

Na

### What are the browser driver(s) and version(s) where you see this issue?

Na

### Are you using Selenium Grid?

No

**Comment by diemol** [comment]
Please try the most recent release (4.9.1).

**Comment by tarunaggarwal23** [comment]
I am getting this error with 4.9.1

module not found: processed.jcommander
module not found: processed.async.httpclient

**Comment by diemol** [comment]
Can you share the complete code to reproduce it? There are also some open issues, please check if that is your case. https://github.com/SeleniumHQ/selenium/milestone/11

**Comment by joerg1985** [comment]
Should be fixed with #12173, in 4.11.0

**Comment by diemol** [comment]
Closing as per @joerg1985's comment.
